### DISCUSSION TITLE
Dependent territories

### DISCUSSION THREAD

**Discussion by nerfologist** [dr5hn/countries-states-cities-database#235] [Ideas]
Hello,
it would be useful to link [dependent territories](https://en.wikipedia.org/wiki/Dependent_territory) to their respective controlling country.

> A dependent territory, dependent area, or dependency is a territory that does not possess full political independence or sovereignty as a sovereign state, yet remains politically outside the controlling state's integral area.

> A dependent territory is commonly distinguished from a country subdivision by being considered not to be a constituent part of a sovereign state. [...]

One such example is [Bouvet Island](https://en.wikipedia.org/wiki/Bouvet_Island). It is represented in the countries list, with iso2 code `BV` and iso3 code `BVT`. Its controlling country is Norway.

**Comment by dr5hn** [comment]
@nerfologist Thanks for the suggestion. 🎉 

Btw, I am a little bit confused, that will these territories be treated as a country or state?

**Comment by nerfologist** [comment]
I'm not 100% sure but I think they are considered as countries (they have ISO-3166 country codes); they just have a connection to another country. I changed the wording in the original issue to clarify this; Bouvet Island is in the list of countries.

PS: Thanks for your super fast answer :-)

**Comment by dr5hn** [comment]
@nerfologist Thanks for the info.

I will check this out soon.
